webui/react/src/utils/types.ts

```diff
@@ -234,3 +235,22 @@ export const tsbMatchesSource = (tensorboard: Command, source: TBSource): boolea
       return false;
   }
 };
+
+export const getMetricValue = (workload?: Workload, metricName?: string): number | undefined => {
```

Comment:
**nit:** `getMetricValueFromWorkload`?

Reply:
IMO it's longer and not immediately clearly better if we have other versions of this we can do something like that?